compiler/lib/flow.ml

```diff
@@ -416,7 +416,7 @@ let the_native_string_of ~target info x =
 let the_block_contents_of info x =
   match the_def_of info x with
   | Some (Block (_, a, _, _)) -> Some a
-  | Some (Prim (Extern "caml_make_array", [ x ])) -> (
+  | Some (Prim (Extern "caml_array_of_uniform_array", [ x ])) -> (
```

Comment:
Don't you need both? Or normalize the primitive in the parser ?

Reply:
I use `Primitive.alias` for that.
https://github.com/ocaml-wasm/wasm_of_ocaml/blob/c5e98de255d38f2fed15d1ad20d276bf9ae36463/compiler/lib-wasm/generate.ml#L1174

Reply:
ok

Reply:
Thinking more about, Are you sure this is enough? I would expect the alias to be resolved late (during code generation)

Reply:
Indeed. I thought this was resolved much sooner.